static/src/javascripts/lib/fastdom-promise.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import fastdom from 'fastdom';
 
-const promisify = fdaction => (fn: Function, ctx: Object) =>
+const promisify = fdaction => (fn: Function, ctx: ?Object) =>
```

Comment:
What does the questionmark add? I should use symbolhound but I'm sure you'll give a clearer explanation 😄